Assets/Script/Gameplay/Visuals/CameraPositioner.cs

```diff
@@ -40,6 +56,12 @@ public void Initialize(CameraPreset preset)
 
             // Rotation
             transform.localRotation = Quaternion.Euler(preset.Rotation, 0f, 0f);
+
+            // Set camera preset
+            _preset = preset;
+
+            // Animate the highway raise
+            _coroutine = StartCoroutine(RaiseHighway(_preset, true));
```

Comment:
Maintainers probably have better ideas about how to do this, but this should be protected by a check that we're actually in gameplay before running the coroutine, otherwise this breaks the preview track in settings. The suggestion works well enough for testing, though.

```suggestion
            if (FindObjectOfType<GameManager>() != null)
            {
                _coroutine = StartCoroutine(RaiseHighway(_preset, true));
            }
```